api/docs/http_report.md

```diff
@@ -55,7 +55,7 @@ example is formatted for readability)
      "job_id"=>null,
      "catalog_uuid"=>"827a74c8-cf98-44da-9ff7-18c5e4bee41e",
      "catalog_format"=>1,
-     "report_format"=>8,
+     "report_format"=>9,
```

Comment:
Just double checking it looks like we bumped to 8 when we added `transaction_completed` in8d9df399356fae8d5d0e667e0d14031470c3a799 released in 5.4.0, so 👍 to bumping here.